docs/user-guide/creating-application/manage-traffic.md

```diff
@@ -0,0 +1,121 @@
+# Manage Traffic
+
+Devtron helps you to manage your **Canary** and **Blue-Green** deployments by providing visibility and easy controls to manage how new versions (releases) are shared with users.
+
+Devtron allows you to:
+
+* Quickly view the current deployment status and detailed progress.
+
+* For canary deployments you can manually trigger the next step for the release or fully complete the rollout.
+
+* For Blue-Green deployments 
```

Comment:
Here the user will have 0 context on let's say blue green, how will they understand all these. I believe the structuring should be.

Deployment strategies --> Discuss on configuration etc, then navigate to app details for visibility